### DISCUSSION TITLE
Accessing GitHub Copilot Usage API with a Business Account – Is it supported?

### DISCUSSION THREAD

**Discussion by rajeshharne** [community/community#144279] [Apps, API and Webhooks] [answered]
### Select Topic Area

Question

### Body

Question: 
Hello everyone,

I’m looking to track GitHub Copilot usage metrics through the API for our organization, which currently has a GitHub Business license. Specifically, I’d like to use the following endpoint:

/enterprises/{enterprise}/copilot/usage

Before diving in, I wanted to confirm: does the GitHub Business license provide access to the Copilot usage API, or is this feature exclusively available for GitHub Enterprise accounts? If it’s enterprise-only, are there any alternative methods for tracking Copilot usage in a Business account?

Thanks for any insights or guidance!

**Comment by veverkap** [comment, marked as answer]
Both the usage API and the new metrics API (https://docs.github.com/en/rest/copilot/copilot-metrics?apiVersion=2022-11-28) only require Copilot Business access.

**Reply by rajeshharne** [reply]
Thanks @veverkap for response, but in the documentaion its mentioend enterprise only. I am trying to use it for business but no luck. could you pleae guide me on how to enable it in business license and consume APIs?  or any links you can share which can be referred. 
Thanks!
RAJESH
